lib/seattleflu/id3c/cli/command/etl/redcap_det_scan.py

```diff
@@ -552,7 +567,7 @@ def create_specimen(record: dict, patient_reference: dict) -> tuple:
 
     note = None
 
```

Comment:
For consistency's sake do you want to add a comment ... is it that not all instruments have the able_to_test variable?

Reply:
Yes. Thank you for pointing this out. It actually reminded me of another change that needs to be added.